authors/settings.py

```diff
@@ -19,7 +19,7 @@
 # See https://docs.djangoproject.com/en/1.11/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = '7pgozr2jn7zs_o%i8id6=rddie!*0f0qy3$oy$(8231i^4*@u3'
+SECRET_KEY = os.environ.get('SECRET_KEY')
```

Comment:
Use `os.environ["SECRET_KEY"]` instead. This will raise an error if the secret key is not provided from the environment variable. When you use `os.environ.get("SECRET_KEY")` it returns **None** when the key is not present

Reply:
Okay. Let me implement these changes.